acceptance/tests/resource/service/puppet_mcollective_service_management.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppet and Mcollective services should be manageable with Puppet"
+
+confine :except, :platform => 'windows'
+confine :except, :platform => 'solaris' # See PUP-5262
```

Comment:
if this is later to be fixed on solaris due to this ticket, perhaps you could use an `expect_failure()` somewhere so that it starts failing when the fix comes in and the fix author is reminded to re-enable this test?